docs/getting-started-with-snowplow-bdp/enterprise/setup-guide-gcp/index.md

```diff
@@ -28,6 +28,8 @@ Please add **techops-cloud-admin@snowplowanalytics.com** to your project with
 - `Roles/monitoring.admin`
 - `Roles/iam.roleAdmin`
 - `Roles/iam.serviceAccountAdmin`
+- `Roles/secretmanager.secretAccessor`
+- `Roles/secretmanager.secretVersionAdder`
```

Comment:
> Only customers who move onto using RDB-Loader in GCP will need to add these.

Should we highlight this here?

Reply:
Agreed, have added qualification for the added roles